src/components/primitives/Download/Download.md

```diff
@@ -13,6 +14,7 @@ import img1 from "./Download.png";
   <Download
     href="https://escolalms.github.io/headless-format/paper.pdf"
     fileName={<strong>myPaper.pdf</strong>}
+    darkTheme
```

Comment:
https://github.com/EscolaLMS/Components/blob/main/src/theme/provider.tsx#L11 

check theme `mode` attribute to see if we're in dark or light mode